### PR TITLE
feat: autofill empty denoms in MsgLeveragedLiquidate

### PR DESCRIPTION
**Pull request by toteki**

### CODE REVIEW

File: x/leverage/keeper/keeper.go [hunk 1]
```diff
@@ -372,6 +372,21 @@ func (k Keeper) Liquidate(
 func (k Keeper) LeveragedLiquidate(
 	ctx sdk.Context, liquidatorAddr, borrowerAddr sdk.AccAddress, repayDenom, rewardDenom string,
 ) (repaid sdk.Coin, reward sdk.Coin, err error) {
+	// If the message did not specify repay or reward denoms, select one arbitrarily (first in
+	// denom alphabetical order) from borrower position. Then proceed normally with the transaction.
```
[the remark this answers is not in the document]
Comment (EbrahimUmee) [reply x/leverage/keeper/keeper.go:376]: Frontend will input the repay and reward denom anyway, so no need to worry for now, in the future we will follow up with a query to make it more predictable

**Review by EbrahimUmee — APPROVED** [review]
LGTM

File: x/leverage/client/cli/tx.go [hunk 2]
```diff
@@ -349,8 +349,14 @@ $ umeed tx leverage lev-liquidate %s uumee uumee --from mykey`,
...
 			}
 
-			repayDenom := args[1]
-			rewardDenom := args[2]
+			var repayDenom, rewardDenom string
+			if len(args) > 1 {
+				repayDenom = args[1]
+			}
+
+			if len(args) > 1 {
```
Comment (toteki) [inline x/leverage/client/cli/tx.go:357]:
```suggestion
			if len(args) > 2 {
```

File: x/leverage/client/cli/tx.go [hunk 3]
```diff
@@ -349,8 +349,14 @@
...
```
[the remark this answers is not in the document]
Comment (toteki) [reply x/leverage/client/cli/tx.go]: fixed

**Pull request merged**